D_controllers.md

```diff
@@ -393,17 +417,18 @@ def index(conn, _params) do
   |> render "index.xml", content: some_xml_content
 end
 ```
+
 We would then need to provide an `index.xml.eex` template which created valid xml, and we would be done.
 
-For a list of valid content mime-types, please see the documentation from the plug middleware framework: [https://github.com/elixir-lang/plug/blob/master/lib/plug/mime.types](https://github.com/elixir-lang/plug/blob/master/lib/plug/mime.types)
+For a list of valid content mime-types, please see the [mime.types](https://github.com/elixir-lang/plug/blob/master/lib/plug/mime.types) documentation from the plug middleware framework.
 
 ### Setting the HTTP Status
 
 We can also set the HTTP status code of a response similarly to the way we set the content type. The `Plug.Conn` module, imported into all controllers, has a `put_status/2` function to do this.
 
-`put_status/2` takes `conn` and either an integer or a "friendly name" used as an atom for the status code we want to set. Here is the list of supported [friendly names](https://github.com/elixir-lang/plug/blob/master/lib/plug/conn/status.ex#L7-L63).
+`put_status/2` takes `conn` as the first parameter and as the second parameter either an integer or a "friendly name" used as an atom for the status code we want to set. Here is the list of supported [friendly names](https://github.com/elixir-lang/plug/blob/master/lib/plug/conn/status.ex#L7-L63).
 
-Let's change the status in our `PageController` `index` action.
+Let's change the status in our `PageController` `index` action in `web/controllers/page_controller.ex`.
```

Comment:
I think we can use the original here. Readers should know where the file is by now. :)